Modules/_zstd/decompressor.c

```diff
@@ -747,16 +741,9 @@ _zstd_ZstdDecompressor_unused_data_get_impl(ZstdDecompressor *self)
 {
     PyObject *ret;
 
-    /* Thread-safe code */
-    Py_BEGIN_CRITICAL_SECTION(self);
-
+    /* Thread-safe code; CRITICAL_SECTION guards are managed by AC */
```

Comment:
I think it's fine not to have this comment. I don't think we have this kind of comments elsewhere.